docs/voiceover-guidelines.md

```diff
@@ -1,61 +1,179 @@
 
-To avoid that problem, the cell needs to become a VoiceOver element itself, with a label that tells the story that is displayed in the cell. A good example is the Reader cell, where its content is read by VoiceOver as follows:
-```
-Post by {author}, from {blog}, {time}.
-{post_title}
-{post_short_description}
+<img src="images/voiceover-guidelines/textfield-empty.png" width="400">
+
+However, a problem shows up if you enter `”John”` as the first name, and `”Doe”` as the last name. 
+
+<img src="images/voiceover-guidelines/textfield-filled.png" width="400">
+
+VoiceOver will no longer read the `”First name”` and `”Last name”` placeholders. How will a user, especially someone with short-term memory, know which field is which? 
+
+You can improve the experience by making sure that the text fields have accessibility labels.
+
+```swift
+firstNameTextField.accessibilityLabel = "First name"
+lastNameTextField.accessibilityLabel = "Last name"
 ```
-This pulls together the most important information from different UILabels into a single well-structured description.
 
-### Setting the spoken order in a complex view
-Some column-based data is contained inside of stack views. VoiceOver may not speak the data in the desired order. For example, this tableview cell has nested stack views. The vertical stack view is the parent and multiple horizontal stack views are children. Two labels are contained inside of each horizontal stack view, one with the title for the data (Subtotal) and one for the data value ($999.99).
+<img src="images/voiceover-guidelines/textfield-filled-with-label.png" width="400">
 
-![](images/voiceover-guidelines.png)
+With this, VoiceOver will read the label (e.g. `"First name"`) and the value (e.g. `"John"`).
 
-The reading order is expected to be spoken as, "Subtotal: nine hundred ninety-nine dollars and ninety nine cents". In this case, VoiceOver defaults to reading the first item in each horizontal stack view, followed by the second. The default order speaks a list of titles: "Subtotal - Discount - Shipping - Taxes" and so on, followed by the values: "$999.99, -$601.00, $0.01, $333.33", making the information difficult to comprehend.
+### <a name="grouping-elements"></a>Grouping Elements
 
-Use `.accessibilityElements` on the parent view to list the desired reading order of the subviews.
+If a group of elements represent a single unit of information, consider grouping them into one accessibility element. This helps reduce clutter and makes your app easier to understand and navigate. 
+
+Take the following custom `UITableViewCell` as an example. It has at least 5 accessible elements. 
+
+<img src="images/voiceover-guidelines/group-elements-before.png" width="320">
+
+Since there are potentially more cells like this in the table, it would be very easy for a VoiceOver user to lose context. To improve this, you can:
+
+- Group the elements together by concatenating the information in the `UITableViewCell`'s `accessibilityLabel`.
```

Comment:
```suggestion
- Group the elements by concatenating the information in the `UITableViewCell`'s `accessibilityLabel`.
```
The "together" here isn't necessary since it creates a tautology 😉

Reply:
TIL **tautology** 😄 🙇